### DISCUSSION TITLE
"npx prisma init" results in "Downloading 0% Error: unable to get local issuer certificate"

### DISCUSSION THREAD

**Discussion by Trixtoparadise** [prisma/prisma#25004] [Q&A]
### Question

Greetings everyone. I need your help. I have been trying to initialize prisma in my project and keeps giving me the error below.
I have tried to run the "npm set strict-ssl false" command. It did not work. Is there any way to initialize manually or bypass this error? 

- Please note that I am using a company machine. Maybe that might be the problem. 

### How to reproduce (optional)

npx prisma init

### Expected behavior (optional)

Download Prisma engines for Node API

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: Windows 11 Pro
- Database: PostgreSQL
- Node.js version: 16.20.2
- 'prisma -v' is giving me the same error

**Comment by ludralph** [comment]
Hi @Trixtoparadise 👋 

Can you try the recommendation in this [closed issue](https://github.com/prisma/prisma/issues/9805)?

**Reply by Trixtoparadise** [reply]
Okay let me look into it.

**Reply by Trixtoparadise** [reply]
HI @ludralph I have tried adding extra node certificates and I have ran into the same issue. Did i go about this the correct way or  did I make a mistake?

![Screenshot (7)](https://github.com/user-attachments/assets/58896eb9-eecb-4d9c-94c3-104e04865b94)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
